confdgnmi/src/confd_gnmi_client.py

```diff
@@ -104,10 +105,15 @@ def generate_subscriptions(subscription_list, poll_interval=0.0,
 
         if subscription_list.mode == gnmi_pb2.SubscriptionList.POLL:
             for i in range(poll_count):
+                # TODO preparation for interactive POLL control
+                # print("Press Enter to Poll")
+                # sys.stdin.readline()
```

Comment:
Do not leave commented-out code, even as a "preparation" (perhaps only when that next step is coming immediately).